rpc/backend/blocks.go

```diff
@@ -347,13 +347,16 @@ func (b *Backend) HeaderByHash(blockHash common.Hash) (*ethtypes.Header, error)
 // BlockBloom query block bloom filter from block results
 func (b *Backend) BlockBloom(blockRes *tmrpctypes.ResultBlockResults) (ethtypes.Bloom, error) {
 	for _, event := range blockRes.EndBlockEvents {
-		if event.Type != evmtypes.EventTypeBlockBloom {
+		if event.Type != evmtypes.NewEventTypeBlockBloom {
 			continue
 		}
-
 		for _, attr := range event.Attributes {
+			val := rpctypes.UnmarshalMessageToString(attr.Value)
+			rawVal := []byte(val)
+			// Fix bloom bytes too big
+			rawVal = bytes.ReplaceAll(rawVal, []byte{239, 191, 189}, []byte{128})
```

Comment:
@yihuang Any better way to handle bloom encode data change? since quote and bracket when emit [TypedEvent](https://github.com/evmos/ethermint/pull/1544/files)